pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -96,8 +96,8 @@ func NewKubeGenericRuntimeManager(
 	cpuCFSQuota bool,
 	runtimeService internalApi.RuntimeService,
 	imageService internalApi.ImageManagerService,
-) (*kubeGenericRuntimeManager, error) {
-	kubeRuntimeManager := &kubeGenericRuntimeManager{
+) (*KubeGenericRuntimeManager, error) {
```

Comment:
In what package will you be testing with `KubeGenericRuntimeManager`? Ideally, other packages should not be aware of a specific implementation of `Runtime`